.github/workflows/staging_pipeline.yml

```diff
@@ -80,9 +82,19 @@ jobs:
           # runs 'bundle install' and caches installed gems automatically
           bundler-cache: true
 
-      - name: Rubocop
+      - name: Set up Node.js
```

Comment:
Need these changes on the production pipeline too I think

Reply:
Have created a separate PR with quality of life improvements for the 2 workflows in #503. Once that’s merged in, will rebase it into this PR and then merge in the specific changes needed to run all linting tasks in the staging workflow.